src/app/services/serviceConfig.js

```diff
@@ -391,9 +378,13 @@ const postServiceConfig = async (req, res) => {
   try {
     const serviceModels = await buildCurrentServiceModel(req);
 
+    // const currentService = serviceModels.currentServiceModel;
```

Comment:
Remove commented out code if it's not going to be used again.

Reply:
Removed.